### PR TITLE
chore: bump deps, remove irrelevant glamorous-native benchmark

### PR DESCRIPTION
**Pull request by headfire94**
update deps with most relevant react-native version.
Had to remove the benchmark with `glamorous-native` cause it doesn't support new RN versions, not relevant anymore.
Removed babel plugins from build process, bundled result works fine with our current metro setup.

## Motivaton

I need this towards next PR where i test react-native-reanimated support, it doesn't work with current deps

### CODE REVIEW

**Review by alexandrius — APPROVED** [review]
utACK. Would be nice to compare with modern technologies like `unistyles` and `nativewind@4`

File: benchmark/results.js [hunk 1]
```diff
@@ -1,6 +1,5 @@
 const results = {
-  "Shakl": 0.023759252519128908,
-  "Emotion": 0.080078398403365,
-  "Glamorous Native": 0.13810396948295817,
-  "Styled Components": 0.15223091570627334
+  "Shakl": 9.832410541863835e-7,
```
Comment (andrejborstnik) [inline benchmark/results.js]: Is this right? Seems like an error?

**Review by andrejborstnik — APPROVED** [review]
utACK

File: benchmark/results.js (hunk 1 above)
Comment (headfire94) [reply benchmark/results.js]: i also was thinking like that, but i can't find where we have error

Comment (headfire94) [reply benchmark/results.js]: found, just a typo https://github.com/ExodusMovement/shakl/pull/12/commits/4c4b908fbc9ac7f59b2d1acd69954c19336eb6b1

**Pull request merged**